DESCRIPTION

```diff
@@ -40,7 +40,7 @@ Suggests:
     httr (>= 1.2.1),
     jsonlite,
     mockery,
-    patrick,
+    patrick (>= 0.2.0),
```

Comment:
this version (>1 year old) lets us get auto-named tests instead of redundantly doing `.test_name = x, param = x` (used in `nested_ifelse_linter()` here)